doc/release-notes-14023.md

```diff
@@ -0,0 +1,8 @@
+Accout API removed
```

Comment:
Typo.

Reply:
FWIW, `codespell` catched this typo. Fixed in the `codespell` PR #13954.

Please review :-)